common/predictive-text/package.json

```diff
@@ -45,6 +45,9 @@
     "typescript": "^3.7.2"
   },
   "dependencies": {
+    "@keymanapp/keyboard-processor": "^14.0.76",
```

Comment:
Why is `keyboard-processor` a dependency here?  `web-utils` was created explicitly to avoid this.

Reply:
Must have gotten in there by mistake (somehow????). Nice catch!